msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/ui/DiscussionForumBean.java

```diff
@@ -540,15 +540,13 @@ public String getOpenDate(){
 	}
 
 	public void setOpenDate(String openDateStr){
-		if (StringUtils.isNoneBlank(openDateStr, openDateISO)) {
+		if (StringUtils.equals(openDateStr, openDateISO)) {
 			try{
 				Date openDate = (Date) datetimeFormat.parse(openDateISO);
 				forum.setOpenDate(openDate);
 			}catch (ParseException e) {
 				log.error("Couldn't convert open date", e);
 			}
-		}else{
-			forum.setOpenDate(null);
```

Comment:
Is it still possible to empty the date to remove it?

Reply:
I thought it was not possible to leave the dates blank, I've restored it!